specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/databases.json

```diff
@@ -200,29 +200,38 @@
         },
         "x-ms-long-running-operation": true,
         "x-ms-examples": {
-          "Creates a database with minimum number of parameters.": {
-            "$ref": "./examples/CreateDatabaseMin.json"
+          "Creates a VCore database by specifying service objective name.": {
+            "$ref": "./examples/CreateVCoreDatabaseByServiceObjective.json"
           },
-          "Creates a VCore database with minimum number of parameters.": {
-            "$ref": "./examples/CreateVCoreDatabase.json"
+          "Creates a VCore database by specifying sku name and capacity.": {
+            "$ref": "./examples/CreateVCoreDatabaseBySkuNameCapacity.json"
           },
-          "Creates a database from restore with database deletion time.": {
-            "$ref": "./examples/CreateDatabaseRestoreMode.json"
+          "Creates a data warehouse by specifying DTU capacity.": {
```

Comment:
We never refer to DTU in our documentation. So creating DW by DTU is not a valid use case for us.